setup.cfg

```diff
@@ -96,7 +96,6 @@ doctest_plus = enabled
 text_file_format = rst
 open_files_ignore = "astropy.log" "/etc/hosts"
 remote_data_strict = true
-addopts = -p no:warnings
```

Comment:
don't yet remove this line, we still have hundreds of warnings from other modules

Reply:
Oops, that shouldn't have been in there!! Removing.